functional-test/src/test/groovy/org/rundeck/tests/functional/api/job/JobExecutionSpec.groovy

```diff
@@ -1876,6 +1876,110 @@ class JobExecutionSpec extends BaseContainer {
                 "</joblist>"
     }
 
+    def "test-renamed-child-job"() {
+        given:
+        def projectName = "test-renamed-child-job"
+        def client = getClient()
+        ObjectMapper mapper = new ObjectMapper()
+        Object projectJsonMap = [
+                "name": projectName.toString(),
+                "description": "test-renamed-child-job",
+                "config": [
+                        "test.property": "test value",
+                        "project.execution.history.cleanup.enabled": "true",
+                        "project.execution.history.cleanup.retention.days": "1",
+                        "project.execution.history.cleanup.batch": "500",
+                        "project.execution.history.cleanup.retention.minimum": "0",
+                        "project.execution.history.cleanup.schedule": "0 0/1 * 1/1 * ? *"
+                ]
+        ]
+
+        def responseProject = createSampleProject(projectJsonMap)
+        assert responseProject.successful
+
+        // Note how the name of the child job iin the ref s Child-2.
+        // This simulates the renaming of the child job.
+        def testXml = "<joblist>\n" +
```

Comment:
FYI, you can use groovy triple-double-quote to make a multi-line string, it might be easier to read., 
e.g.
```groovy
def testXml = """
..etc..
"""
```